### PR TITLE
Updating the otel-demo config & doc following onboarding issues

### PR DESCRIPTION
**Pull request by mistermocha**
* Updated otel-collector to 0.72
* Updated otelcol-config.yml to work right with the new collector version
* Updated docs to be hopefully more helpful

### CODE REVIEW

**Review by oded-dd — APPROVED** [review]

File: otel-agent/examples/otel-demo-docker-compose/README.md [hunk 1]
```diff
@@ -15,18 +19,35 @@ All the examples are using `open-telemtry` SDK. Which can be found [here](https:
...
+   * The `CORALOGIX_API_KEY` should come from the integration link indicated above. Use the "Send Your Data"
+     access key.
+2. Run `docker compose up -d`
+3. In your Coralogix UI, visit the "Explore" tab and click on "Tracing". You should see some input.
+   * If you do not, check out `docker logs otel-col` and look for any issues. It should look like this
+   on the latest line of the log:
 ```
+2023-05-01T22:39:28.349Z	info	service/service.go:157	Everything is ready. Begin running and processing data.
+```
+4. Visit http://localhost:8089 and try increasing the volume of traffic to increase the inputto Coralogix.
```
Comment (ShauliSolomovich) [inline otel-agent/examples/otel-demo-docker-compose/README.md:47]: just missing a space here `inputto -> input to`
other than that looks great !

**Review by ShauliSolomovich — CHANGES_REQUESTED** [review]

**Comment by nicolastakashi** [comment]
@mistermocha can you review the conflicts?

File: otel-agent/examples/otel-demo-docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
 
   # Collector
   otelcol:
-    image: otel/opentelemetry-collector-contrib:0.56.0
+    image: otel/opentelemetry-collector-contrib:0.72.0
```
Comment (matej-g) [inline otel-agent/examples/otel-demo-docker-compose/docker-compose.yml:25]: Why not latest release?

File: otel-agent/examples/otel-demo-docker-compose/README.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # OpenTelemetry Demo
-This Online Boutique with exmaple how to confiugre the open-telemetry collector to send data to Coralogix.
+
+The Demo stack here shows how the open-telemetry collector sends data to Coralogix. The stack here serves
+a robust online store with an automatic traffic generator. The basics will send tracing data to be read
+by Coralogix.
```
Comment (matej-g) [inline otel-agent/examples/otel-demo-docker-compose/README.md:5]: Few stylistic suggestions:
```suggestion
This demo shows how the OpenTelemtry collector sends data to Coralogix. The stack here serves
a robust online store with an automatic traffic generator. The signals data will be sent to Coralogix backend.
```

File: otel-agent/examples/otel-demo-docker-compose/README.md [hunk 4]
```diff
@@ -15,18 +19,35 @@ All the examples are using `open-telemtry` SDK. Which can be found [here](https:
 
 The source code for this demo is available [here](https://github.com/open-telemetry/opentelemetry-demo/tree/v0.3.1-alpha) under the `/src` directory. 
 
+## Before you begin
 
+1. Install the Docker client on your machine. This demo runs in Docker.
+2. Visit https://dashboard.eu2.coralogix.com/#/signup and sign up for an account. 
+3. Visit `https://YOUR_ACCOUNT_NAME.app.coralogix.us/#/integration/apikey` (updating `YOUR_ACCOUNT_NAME`
+   with the name of the account you had created at signup. This is where you can select an API key. 
+   Keep this page around for one moment.
```
Comment (matej-g) [inline otel-agent/examples/otel-demo-docker-compose/README.md:28]:
```suggestion
2. [Sign up](https://dashboard.eu2.coralogix.com/#/signup) for a Coralogix account, if you don't have one yet. 
3. Visit `https://YOUR_ACCOUNT_NAME.app.coralogix.us/#/integration/apikey` (updating `YOUR_ACCOUNT_NAME`
   with the name of the account you have created during the signup. This is where you can access your API key. 
   Keep this page around for one moment.
```

File: otel-agent/examples/otel-demo-docker-compose/README.md [hunk 5]
```diff
@@ -15,18 +19,35 @@ All the examples are using `open-telemtry` SDK. Which can be found [here](https:
...
-```bash
-docker-compose up -d 
+Getting started is easy! 
+
+1. Open the [otelcol-config.yml](https://github.com/coralogix/telemetry-shippers/blob/master/otel-agent/examples/otel-demo-docker-compose/otelcol-config.yml)
+   file in this directory and update the two coralogix fields at the top of the page.
+   * The `CORALOGIX_ENDPOINT` should be the appropriate open-telemetry routes. Please Check
+     https://coralogix.com/docs/coralogix-endpoints/ for the most appropriate endpoint.
+   * The `CORALOGIX_API_KEY` should come from the integration link indicated above. Use the "Send Your Data"
+     access key.
```
Comment (matej-g) [inline otel-agent/examples/otel-demo-docker-compose/README.md:39]:
```suggestion
   file in this directory and update the two Coralogix fields at the top of the page.
   * The `CORALOGIX_ENDPOINT` should be set to appropriate endpoints accepting data. Please see [our documentation](https://coralogix.com/docs/coralogix-endpoints/) to know which endpoint to use. 
   * The `CORALOGIX_API_KEY` should come from the integration link indicated above. Use the "Send Your Data"
     access key.
```

File: otel-agent/examples/otel-demo-docker-compose/README.md [hunk 6]
```diff
@@ -15,18 +19,35 @@ All the examples are using `open-telemtry` SDK. Which can be found [here](https:
...
+2. Run `docker compose up -d`
+3. In your Coralogix UI, visit the "Explore" tab and click on "Tracing". You should see some input.
+   * If you do not, check out `docker logs otel-col` and look for any issues. It should look like this
+   on the latest line of the log:
 ```
+2023-05-01T22:39:28.349Z	info	service/service.go:157	Everything is ready. Begin running and processing data.
+```
+4. Visit http://localhost:8089 and try increasing the volume of traffic to increase the input to Coralogix.
```
Comment (matej-g) [inline otel-agent/examples/otel-demo-docker-compose/README.md:47]:
```suggestion
4. Visit `http://localhost:8089` and try increasing the volume of traffic to increase the output to Coralogix.
```

**Review by matej-g — COMMENTED** [review]
Looks nice 👍, some stylistic suggestions.

File: otel-agent/examples/otel-demo-docker-compose/docker-compose.yml (hunk 2 above)
Comment (mistermocha) [reply otel-agent/examples/otel-demo-docker-compose/docker-compose.yml:25]: I stand corrected! I was able to update to newer versions!

Comment (mistermocha) [reply otel-agent/examples/otel-demo-docker-compose/docker-compose.yml:25]: 0.75 is the latest image, even though newest source code is 0.76. Confirmed through testing

```                                                                                                                                              1.6s
Error response from daemon: manifest for otel/opentelemetry-collector-contrib:0.76.0 not found: manifest unknown: manifest unknown
```

Comment (matej-g) [reply otel-agent/examples/otel-demo-docker-compose/docker-compose.yml:25]: There were some issues with the latest release AFAIK, the correct image should be `0.76.1`.

**Comment by mistermocha** [comment]
Oh goodness, things got messy with my diff during a rebase. I'm dropping this and starting over.

**Pull request closed without merging**